src/course-page/CoursePage.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCog } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@edx/paragon';
+
+function CoursePage({
```

Comment:
I think this component name is a bit misleading. A better name I feel would be: `CoursePageConfigTile`.

Reply:
I believe these are generally called "Cards" by both Bootstrap and the edX UX team: https://getbootstrap.com/docs/4.0/components/card/

Reply:
Should I rename the file as well? What about the directory? The former seems very reasonable, but I am unsure about the latter.

Reply:
I think it's a virtue to keep naming consistent... so if we think these are 'cards' instead of 'pages', then I'd consistently change the term, yeah.  Assuming that the course-page module is all about code related to these cards.

Reply:
It seems to be about the cards, and I'll go with that naming convention. Thank you.

Reply:
@taniwha As mentioned above in the comment, the convention here is to call these UI elements Cards, not Tiles. So could you rename this accordingly?